docs/sources/reference/api/docker_remote_api.md

```diff
@@ -68,6 +68,12 @@ New endpoint to rename a container `id` to a new name.
 (`ReadonlyRootfs`) can be passed in the host config to mount the container's
 root filesystem as read only.
 
+`GET /containers/(id)/stats`
+
+**New!**
+This endpoint returns a stream of container stats based on resource usage.
```

Comment:
clarify to "This endpoint returns a live stream of a container's resource usage statistics."